src/app.py

```diff
@@ -3130,6 +3130,7 @@ def _notification_is_dismissed(name, settings):
 
 
 def _filter_and_hydrate_notifications(notifications, current_version=None, data={}):
+    current_version=str(current_version)
     def is_version_more_recent_than_current_version(name):
         # Boring code to handle the fact that "0.1 < 9999~ynh1" is False
```

Comment:
```suggestion
    def is_version_more_recent_than_current_version(name):
        current_version=str(current_version)
        # Boring code to handle the fact that "0.1 < 9999~ynh1" is False
```

Reply:
Cant do that in the general scope because we want to check `if current_version is None` later :sweat_smile: